blockchains/smartchain/assets/0x0005fd45281d89042965acbaf645ecc86bc5ec5c/info.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "1DIAMOND",
+    "description": "1DIAMOND is the currency that one will need to use the services of the incoming ROBOT ARMY automated copy trading platform",
+    "explorer": "https://bscscan.com/token/0x0005fd45281d89042965acbaf645ecc86bc5ec5c",
```

Comment:
should be https://bscscan.com/token/0x0005Fd45281d89042965aCBAf645ecC86bC5Ec5c
will fix later